connectors/salesforce/src/main/java/com/vmware/connectors/salesforce/SalesforceController.java

```diff
@@ -64,7 +45,7 @@
 import static org.springframework.web.util.UriComponentsBuilder.fromHttpUrl;
 
 @RestController
-@SuppressWarnings("PMD.NcssCount")
+@SuppressWarnings({"PMD.ShortVariable", "PMD.NcssCount"})
```

Comment:
Why can't the NCSS violation be fixed?

Reply:
At the time, I was hesitant to rewrite the salesforce controller because I didn't really know how to test it.  I can revisit it now that I do know how: https://github.com/vmware/connectors-workspace-one/pull/4#discussion_r158853726

Reply:
Fixed the NcssCount violation..